samples/msal-node-samples/managed-identity/README.md

```diff
@@ -0,0 +1,53 @@
+# Managed Identity for Azure VM Sample
```

Comment:
Let's move away from samples in the MSAL repo. Can we please have a sample here: https://github.com/Azure-Samples/msal-managed-identity/tree/main/src